garnbarn_api/views.py

```diff
@@ -82,7 +87,8 @@ class TagViewset(viewsets.ModelViewSet):
     serializer_class = TagSerializer
 
     def get_queryset(self):
-        tag = Tag.objects.get_queryset().order_by('id')
+        user_data = self.request.user.get_json_data()["uid"]
```

Comment:
Maybe this can be minimized by `self.request.user.uid`

Reply:
Done in commit bc03e83.